### PR TITLE
RM8791: interco is readonly when invoices and sale/purchaseOrders are…

### PR DESCRIPTION
**Pull request by dtr-axelor**
… at the end of the process.

### CODE REVIEW

File: axelor-account/src/main/resources/views/Invoice.xml [hunk 1]
```diff
@@ -873,6 +873,8 @@
 				   expr="eval: createdByInterco || !partner || __repo__(Company).all().filter('self.partner = ?', partner).fetchOne() == null"/>
 		<attribute for="interco" name="value" expr="eval: false"
 				   if="eval: createdByInterco || !partner || __repo__(Company).all().filter('self.partner = ?', partner).fetchOne() == null"/>
+	   <attribute for="interco" name="readonly" expr="eval: true"
+				   if="eval: statusSelect > 2"/>
```
Comment (gdu-axelor) [inline axelor-account/src/main/resources/views/Invoice.xml]: Replace '>' per '&gt;'

File: axelor-supplychain/src/main/resources/views/PurchaseOrder.xml [hunk 2]
```diff
@@ -25,6 +25,8 @@
 				   expr="eval: createdByInterco || !supplierPartner || __repo__(Company).all().filter('self.partner = ?', supplierPartner).fetchOne() == null"/>
 		<attribute for="interco" name="value" expr="eval: false"
 				   if="eval: createdByInterco || !supplierPartner || __repo__(Company).all().filter('self.partner = ?', supplierPartner).fetchOne() == null"/>
+		<attribute for="interco" name="readonly" expr="eval: true"
+				   if="eval: statusSelect > 3"/>
```
Comment (gdu-axelor) [inline axelor-supplychain/src/main/resources/views/PurchaseOrder.xml]: Replace '>' per '&gt;'

File: axelor-supplychain/src/main/resources/views/SaleOrder.xml [hunk 3]
```diff
@@ -99,6 +99,8 @@
 				   expr="eval: createdByInterco || !clientPartner || __repo__(Company).all().filter('self.partner = ?', clientPartner).fetchOne() == null"/>
 		<attribute for="interco" name="value" expr="eval: false"
 				   if="eval: createdByInterco || !clientPartner || __repo__(Company).all().filter('self.partner = ?', clientPartner).fetchOne() == null"/>
+		<attribute for="interco" name="readonly" expr="eval: true"
+				   if="eval: statusSelect > 3"/>
```
Comment (gdu-axelor) [inline axelor-supplychain/src/main/resources/views/SaleOrder.xml]: Replace '>' per '&gt;'

**Comment by dtr-axelor** [comment]
Requested changes applied.

**Pull request merged**